.github/workflows/auto_labeler.yml

```diff
@@ -24,3 +24,7 @@ jobs:
               repo: context.repo.repo,
               labels: ['Status: Needs Triage']
             });
+      - name: Apply Type Label
+        uses: actions/labeler@v2
+        with:
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
This workflow does not run on new pull requests, only issues, so this won't work and it needs a separate workflow with `pull_request_target` event.

Reply:
https://github.com/actions/labeler/issues/36
This issue documents jack's comment